<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>0</y>
       <width>211</width>
       <height>55</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ink Free</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Report Generator</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="search_B">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>3</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#search_B{
  font-weight: 600;
  border-radius: 10px;
  border: 1px solid black;
  padding: 5px 15px;
  margin-top: 10px;
  outline: 0px;
}</string>
     </property>
     <property name="text">
      <string>                 Search .csv and Exel files....</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/icons/search.png</normaloff>images/icons/search.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>17</width>
       <height>19</height>
      </size>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>50</y>
       <width>1500</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1001</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Dir">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>15</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>21</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(95, 95, 95);</string>
     </property>
     <property name="text">
      <string>No Files selected....</string>
     </property>
    </widget>
    <widget class="QPushButton" name="B_view">
     <property name="geometry">
      <rect>
       <x>824</x>
       <y>16</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>View report</string>
     </property>
    </widget>
    <widget class="QPushButton" name="B_refresh">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>16</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(212, 212, 212);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/icons/reload.png</normaloff>images/icons/reload.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>19</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="B_export">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>16</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(212, 212, 212);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>images/icons/save.png</normaloff>images/icons/save.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>19</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="Dir_date">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>50</y>
       <width>461</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>No Files selected....</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>385</y>
      <width>271</width>
      <height>197</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(95, 95, 95);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>11</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>42</y>
       <width>141</width>
       <height>144</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QCheckBox" name="check_table">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Table</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_bar">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Bar Chart</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_line">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Line Chart</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_pie">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Pie Chart</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_scatter">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Scatter Plot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="check_lists">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Listings</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>10</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Configure</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>585</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(95, 95, 95);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>4</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(212, 212, 212, 0);</string>
     </property>
     <property name="text">
      <string>Design</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Clr">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>70</y>
       <width>103</width>
       <height>23</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(212, 212, 212);</string>
     </property>
     <item>
      <property name="text">
       <string>Vivid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Greyscale</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>43</y>
       <width>85</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Orientation  :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>70</y>
       <width>84</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Color mode :</string>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>24</y>
       <width>131</width>
       <height>11</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Ori">
     <property name="geometry">
      <rect>
       <x>132</x>
       <y>43</y>
       <width>103</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(212, 212, 212);</string>
     </property>
     <item>
      <property name="text">
       <string>Portrait</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Landscape</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QListWidget" name="list_Details">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>149</y>
      <width>271</width>
      <height>144</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="list_ToSelect">
    <property name="geometry">
     <rect>
      <x>307</x>
      <y>150</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="list_Selected">
    <property name="geometry">
     <rect>
      <x>663</x>
      <y>150</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>304</y>
      <width>271</width>
      <height>77</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(95, 95, 95);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(212, 212, 212, 0);</string>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>23</y>
       <width>131</width>
       <height>11</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="customMode">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>43</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Custom</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="defaultMode">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>43</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="B_removeFromlist">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>212</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/icons/left-arrow.png</normaloff>images/icons/left-arrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="B_addTolist">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>260</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/icons/right-arrow.png</normaloff>images/icons/right-arrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QListWidget" name="list_pref">
    <property name="geometry">
     <rect>
      <x>308</x>
      <y>400</y>
      <width>675</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="B__add">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_cleat">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_clearAll">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>650</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CLEAR ALL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="B_ok">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>650</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
